goty-server/src/main/kotlin/com/aleinin/goty/submit/SubmissionRequest.kt

```diff
@@ -1,27 +1,9 @@
 package com.aleinin.goty.submit
 
-import org.springframework.data.annotation.Id
-import java.util.UUID
-
 data class SubmissionRequest(
```

Comment:
With these changes, what does a SubmissionRequest now represent?

Reply:
A `SubmissionRequest` represents the request body of the `/submissions` endpoint. It's probably overkill for this small of a project, but best practices occasionally recommend using separate data models for REST requests, responses, and your database models. This has the advantage of making the API a little clearer to clients because every field on the `SubmissionRequest` model has a purpose. The client should never supply `id`, `enteredOn`, or `updatedOn` and since there is a specialized request model for that purpose we don't even need to check for that edge case.